packages/sources/moore-hk/src/endpoint/trueusd.ts

```diff
@@ -0,0 +1,33 @@
+import { AdapterEndpoint } from '@chainlink/external-adapter-framework/adapter'
+import { InputParameters } from '@chainlink/external-adapter-framework/validation'
+import { SingleNumberResultResponse } from '@chainlink/external-adapter-framework/util'
+import { config } from '../config'
+import { httpTransport } from '../transport/trueusd'
+
+export const inputParameters = new InputParameters(
+  {
+    field: {
+      default: 'totalTrust',
+      description:
+        'The object-path string to parse a single `result` value. When not provided the entire response will be provided.',
```

Comment:
This description is probably outdated since we have a default for this field.
```suggestion
        'The object-path string to parse a single `result` value.',
```